### PR TITLE
(version 0.13.1) add abs.yaml for 3.13 for pandas

### PR DESCRIPTION
**Pull request by JamesRobertsonGames**
meson-python 

**Destination channel:** defaults

### Links

- Relevant dependency PRs:
  - https://github.com/AnacondaRecipes/pandas-feedstock/pull/46

### Explanation of changes:
- Added abs.yaml to build 3.13 for pandas specific version lock

### CODE REVIEW

**Review by ELundby45 — APPROVED** [review]
LGTM. Note, there is a v0.13.2 if that would be preferred.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -12,13 +12,13 @@ source:
...
 requirements:
   host:
-    - meson 1.0.1
+    - meson >=0.63.3
     - pip
-    - pyproject-metadata 0.7.1
+    - pyproject-metadata >=0.7.1
     - python
     - setuptools
     - tomli 1.2.2  # [py<311]
```
Comment (danpetry) [inline recipe/meta.yaml]:
```suggestion
    - tomli >=1.0.0  # [py<311]
```

Comment (danpetry) [reply recipe/meta.yaml]: the run requirement needs a py<311 selector too

**Review by danpetry — APPROVED** [review]
with changes

File: recipe/meta.yaml (hunk 1 above)
Comment (cbouss) [reply recipe/meta.yaml]: addressed

**Pull request merged**